specification/semantic_conventions.md

```diff
@@ -27,3 +28,41 @@ All Splunk distributions of OpenTelemetry,
 | Attribute  | Type | Description  | Examples  | Required |
 |---|---|---|---|---|
 | `splunk.distro.version` | string | The version number of the Splunk distribution being used. | `1.5.0` | Yes |
+
+Note: this section does not apply to Real User Monitoring libraries, as they do not use the OpenTelemetry Resource.
+
+## Real User Monitoring Spans and Attributes
+
+Real User Monitoring (RUM) libraries MUST set the `service.name` resource attribute to the value of the `applicationName`
```

Comment:
We need to revert back from `serviceName` to `applicationName` or update this.

Reply:
Changed to `applicationName` everywhere, thanks!